src/main/java/session/FormEntrySession.java

```diff
@@ -93,17 +120,14 @@ public void initialize(boolean newInstance, Map<String, String> sessionData) thr
         formDef.initialize(newInstance, sessionWrapper.getIIF());
     }
 
-    public void initialize(boolean newInstance) throws Exception {
-        initialize(newInstance, null);
-    }
-
     private FormDef parseFormDef(String formXml) throws IOException {
         XFormParser mParser = new XFormParser(new StringReader(formXml));
         return mParser.parse();
     }
 
     public String getInstanceXml() throws IOException {
-        return XmlUtil.getPrettyXml(new XFormSerializingVisitor().serializeInstance(formDef.getInstance()));
+        byte[] bytes = new XFormSerializingVisitor().serializeInstance(formDef.getInstance());
+        return new String(bytes, "US-ASCII");
```

Comment:
add a unicode character to a form instance. so you could potentially answer a question with something here: http://unicode-table.com/en/